mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+
```

Comment:
you can add `ignore_missing_imports` here to apply globally

normally I will just add to the `pyproject.toml` file like [this](https://mypy.readthedocs.io/en/stable/config_file.html#example-pyproject-toml) so I can apply the same template across projects

```
[[tool.mypy.overrides]]
module = "*.*"
ignore_missing_imports = true
```